meson_options.txt

```diff
@@ -278,9 +278,9 @@ option('dbus', type : 'combo', choices : ['auto', 'true', 'false'],
 
 option('gnu-efi', type : 'combo', choices : ['auto', 'true', 'false'],
        description : 'gnu-efi support for sd-boot')
-option('efi-cc', type : 'string', value : 'gcc',
+option('efi-cc', type : 'string', value : '$cc',
        description : 'the compiler to use for EFI modules')
-option('efi-ld', type : 'string', value : 'ld',
+option('efi-ld', type : 'string', value : '$ld',
```

Comment:
I don't think we need to use the canary values. Above there's various options which just don't set the default, for example `quotaon-path`. I would prefer to keep this consistent. Also I think not having the default is better because the canary value can be confused with an environment variable.